test/test_variational.py

```diff
@@ -84,7 +84,8 @@ def test_variables() -> None:
     # pylint: disable=C0103
     stan = os.path.join(DATAFILES_PATH, 'variational', 'eta_should_be_big.stan')
     model = CmdStanModel(stan_file=stan)
-    variational = model.variational(algorithm='meanfield', seed=999999)
+    variational = model.variational(algorithm='meanfield', seed=999999,
+                                    output_samples=999)
```

Comment:
I've added `output_samples=999` for explicitness and in case the default number of variational samples changes in the future.